OONIProbeUnitTests/APITest.m

```diff
@@ -30,7 +30,7 @@ - (void)tearDown {
 }
 
 /* This test will get a measurement detail then try to download the related json */
-- (void)testExisting {
+- (void)skipped_testExisting {
```

Comment:
Can we skip conditionally if `CI` is set to `true` [1] using `XCTSkipIf` [2] ?

Also, I think it's important to describe somewhere why we are skipping these tests. It could be a comment in the code describing the problem. Or the comment could just link to an issue (even a closed issue is fine) describing the problem.

[1] https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

[2] https://developer.apple.com/documentation/xctest/xctskipif?language=objc